gr-uhd/examples/grc/rfnoc_images/x310_image_core.grc

```diff
@@ -1,34 +1,37 @@
 options:
```

Comment:
This is the same GRC file as before, only with a different output language. However, that means new generate options, which explains the diff.

![x310_image_core](https://github.com/user-attachments/assets/11da97e3-b492-475e-adba-37524c27b53a)